external/kafka/src/main/scala/org/apache/gearpump/streaming/kafka/lib/source/consumer/FetchThread.scala

```diff
@@ -133,19 +133,21 @@ private[kafka] class FetchThread(
 
   /**
    * fetch message from each TopicAndPartition in a round-robin way
+   *
+   * @return whether to fetch more messages
```

Comment:
akka-streams has this GraphStage called InterLeave where upstreams are cycled and N messages are pulled from each one. I'm wondering if we should expose a policy related to fetch messages which is now round-robin.